Clarify software requirements of Terraform Enterprise (#1980)
hashicorp/terraform-website: content/source/docs/enterprise/before-installing/index.html.md

### Software Requirements (Standalone Deployment)

The following list provides the general requirements of Terraform
Enterprise:

- A 64-bit architecture Linux-based operating system
- Linux Kernel 3.10 or greater
- Docker Engine 17.06.2-ce to 20.10:
  - Docker Engine 18.01.0-ce or greater requires Replicated 2.32.0
    or greater
  - In Online mode, the installer will install Docker Engine
    automatically
  - In Airgapped mode, Docker Engine must be installed before running
    the installer

> - The faccessat2 syscall:

Reviewer: is it worth to direct customers the reason for this requirement like in the release notes?
https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2

I had no idea what the faccessat2 syscall is/does.

Author: How does https://github.com/hashicorp/terraform-website/pull/1980/commits/4816fba670c6b420c4185f866d7dd08fff6417ba look?

### Software Requirements (Standalone Deployment)

> The following list provides the general requirements of Terraform

Reviewer 2: ```suggestion
Every Terraform Enterprise installation requires the following:
```

Enterprise:

- A 64-bit architecture Linux-based operating system
- Linux Kernel 3.10 or greater
- Docker Engine 17.06.2-ce to 20.10:
  - Docker Engine 18.01.0-ce or greater requires Replicated 2.32.0
    or greater
  - Support for [Alpine 3.14](https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2)
    Docker images:
    - Docker Engine 17.06.2-ce to 20.09 requires runc v1.0.0-rc93
    - Docker Engine 20.10 or greater requires libseccomp 2.4.4
  - In Online mode, the installer will install Docker Engine
    automatically

>   - In Airgapped mode, Docker Engine must be installed before running

Reviewer 2: ```suggestion
  - In Airgapped mode, you must install Docker Engine before running
```

    the installer

> The use of some operating systems comes with additional requirements:

Reviewer 2: ```suggestion
Some operating systems also have additional requirements:
```

- [RedHat Enterprise Linux (RHEL) Requirements](./rhel-requirements.html)
- [CentOS Requirements](./centos-requirements.html)

> For other Linux distributions, check Docker requirements:

Reviewer 3: ```suggestion
Other Linux distributions require:
```

> - A 64-bit architecture

Reviewer 3: This being in the "Other" section makes it read like it's not applicable to CentOS/RHEL. Perhaps we can reorder?

Author: Fixed by https://github.com/hashicorp/terraform-website/pull/1980/commits/e03fb3294eee916afe42220e21ba38586b0036db.
